chains/lumia-testnet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "lumia-testnet",
```

Comment:
Wouldn't `lumia-sepolia-testnet` be a better alternative? It appears that, as mentioned [here](https://docs.lumia.org/build/build-environment/rpc#public-endpoints), transaction hashes are currently sent to `ethereum-sepolia-testnet`. In the future, there may be a new `lumia-holeksy-testnet` that will route hashes to `ethereum-holesky-testnet`.